easybuild/tools/systemtools.py

```
@@ -219,30 +222,40 @@ def get_cpu_speed():
         try:
```

The affinity mask is a POSIX thing right? So why not try the BSD thing and then fall back to the affinity mask?